__tests__/costumes.test.js

```diff
@@ -153,7 +73,7 @@ describe("createCostume adapter", () => {
             SELECT * FROM costumes WHERE name='buttless chaps';
         `);
         
-        expect(matchesCostumeInDatabase(buttlessChaps, chapsFromDatabase)).toBe(true);
+        expect(matchesDatabase(buttlessChaps, chapsFromDatabase)).toBe(true);
```

Comment:
Avoid assertions in after all - assertions should happen in each test

Reply:
Do you mean in an afterAll call? I am not sure which assertions you are referring to. This is the code I have for the afterAll call (lines 32-35):
```
// Disconnect from postgres database after all tests done
afterAll(async () => {
    await pool.end()
})

Reply:
Oh sorry - GitHub was hiding some lines, so it looked like it had an `expect` in the `afterAll`